src/main/java/org/thoughtslive/jenkins/plugins/jira/steps/WaitForIssueUpdateStep.java

```diff
@@ -0,0 +1,148 @@
+package org.thoughtslive.jenkins.plugins.jira.steps;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.logging.Logger;
+
+import org.apache.commons.lang.ArrayUtils;
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+import org.jenkinsci.plugins.workflow.steps.Step;
+import org.jenkinsci.plugins.workflow.steps.StepContext;
+import org.jenkinsci.plugins.workflow.steps.StepExecution;
+import org.jenkinsci.plugins.workflow.support.actions.PauseAction;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.thoughtslive.jenkins.plugins.jira.util.JiraStepDescriptorImpl;
+import org.thoughtslive.jenkins.plugins.jira.webhook.JiraIssueChangeListener;
+import org.thoughtslive.jenkins.plugins.jira.webhook.JiraWebHook;
+
+import com.google.common.collect.ImmutableSet;

```

Comment:
Lets apply the google style guide, this line should be sorted to be on top, applies through out.